modules/gdscript/gdscript.cpp

```diff
@@ -1627,15 +1642,22 @@ bool GDScriptInstance::get(const StringName &p_name, Variant &r_ret) const {
 	{
 		HashMap<StringName, GDScript::MemberInfo>::ConstIterator E = script->member_indices.find(p_name);
 		if (E) {
-			if (E->value.getter) {
-				Callable::CallError err;
-				r_ret = const_cast<GDScriptInstance *>(this)->callp(E->value.getter, nullptr, 0, err);
-				if (err.error == Callable::CallError::CALL_OK) {
+			const GDScript::MemberInfo *member = &E->value;
+			if (member->getter) {
+				Callable::CallError ce;
+				if (member->getter_with_name) {
+					Variant property_name = p_name;
+					const Variant *args = &property_name;
+					r_ret = const_cast<GDScriptInstance *>(this)->callp(member->getter, &args, 1, ce);
+				} else {
+					r_ret = const_cast<GDScriptInstance *>(this)->callp(member->getter, nullptr, 0, ce);
+				}
+				if (ce.error == Callable::CallError::CALL_OK) {
 					return true;
 				}
```

Comment:
```suggestion
				return ce.error == Callable::CallError::CALL_OK;
```

I think it's more correct and above similar code. There is a code with the same error(?) below.

Reply:
> Hmm, your change makes sense to me, but I haven't thought too much about what returning false here would mean.

I haven't tested this, but I think there will be an error in the console if it returns `false`. In the current version, if the getter call fails, execution will continue and the raw value will be returned. I'm not sure if this is intentional.

> I've seen this pattern elsewhere in nearby code

> is it okay to do it in this PR or should it be separated?

Yes, other places should also be corrected. I think it can be added to this PR as it's a small change and not a big issue.

Reply:
I'm not so sure what the new behavior would be. There's a few cases where the return value of `get` is used, for instance: https://github.com/godotengine/godot/blob/cdd2313ba27d0a2600a18e849b4c5d1fd6a6e351/core/object/object.cpp#L456-L478. My thinking would be that this is enough of a behavior change that it makes sense to separate it, if only to make testing easier. However, you have much more experience on this than I do, so I'm happy to follow your lead :)

Reply:
If you have concerns about it, you don't have to fix it. I just drew attention to a possible error in the affected method.

Reply:
Sounds good. I'll leave it for another PR then.

In making these changes, I figured it would be cleaner to write a helper function like `call_getter/setter` to reduce code duplication. Sound reasonable?